docs/e2e_test_users.md

```diff
@@ -0,0 +1,28 @@
+# E2E Test users
+
+# Define a test user
+
+Go to the admin interface to `/admin/e2e_tests/testuser/` to define a test user.
+
+# Generate a token
+
+Using the admin graphql endpoint execute the follwing query:
+
+GraphQL mutation:
+```graphql
+mutation {
+  generateTestUserToken(input: { userEmail: "[test user email]"}) {
+    errors
+    token
+  }
+}
+```
+
+Curl command:
+```bash
+curl '[host]/graphql/admin' \
+  -H 'cookie: sessionid=[admin session ID]' \
```

Comment:
question: where do you get the session ID?

Reply:
@paulschreiber I added the details on how to get it